src/main/tables/108_user_instant_launches.sql

```diff
@@ -0,0 +1,15 @@
+SET search_path = public, pg_catalog;
+
+CREATE TABLE IF NOT EXISTS user_instant_launches (
+    id UUID UNIQUE NOT NULL DEFAULT uuid_generate_v1(),
+
+    user_id UUID UNIQUE NOT NULL,
+
+    instant_launches JSONB NOT NULL DEFAULT '{}'::JSONB,
```

Comment:
A comment might be useful on this column too.

Reply:
Comment added.